cypress_test/integration_tests/desktop/impress/annotation_spec.js

```diff
@@ -13,6 +13,8 @@ describe('Annotation Tests', function() {
 
 		if (Cypress.env('INTEGRATION') === 'nextcloud') {
 			desktopHelper.hideSidebarIfVisible();
+		} else if (Cypress.env('USER_INTERFACE') === 'notebookbar') {
+			cy.get('#ModifyPage').click();
```

Comment:
This seems like a bug can you confirm please. Modify page should not work as sidebar button.

Reply:
I don't think it is a bug : )